### PR TITLE
@starsirius => Channel articles for APU RSS feed and avoid vanity fetches

### PR DESCRIPTION
**Pull request by kanaabe**
Closes: https://github.com/artsy/positron/issues/817

This PR replaces the Section fetch with a Channel fetch for the Artsy Partner Updates RSS feed.

I also noticed that we're hitting Positron's API pretty hard with trials to vanity-type URLS like /gagosian and /:id . "Sections" are now deprecated (only one left is Venice Biennale) and Team Blogs have to be manually created anyway (and not often). With that in place, we should instead just make the list of team blogs configurable in a `TEAM_BLOGS` string, and stop doing trials to `/sections/:id` and `/channels/:id` on Positron's API.

### CODE REVIEW

**Comment by kanaabe** [comment]
@starsirius bump!

File: config.coffee [hunk 1]
```diff
@@ -111,6 +111,8 @@ module.exports =
   LOGGER_FORMAT: 'combined'
   GALLERY_INSIGHTS_CHANNEL: '5759e4a6b5989e6f98f77995'
   EDITORIAL_CTA_BANNER_IMG: 'http://files.artsy.net/images/iphone_email.png'
+  ARTSY_PARTNER_UPDATES_CHANNEL: '5762d454b5989e6f98f7799a'
```
Comment (starsirius) [inline config.coffee]: Do we also want to add this env variable to Heroku?

File: config.coffee [hunk 2]
```diff
@@ -111,6 +111,8 @@ module.exports =
...
+  TEAM_BLOGS: '^\/life-at-artsy$|^\/artsy-education$|^\/gallery-insights$|^\/artsy-partner-updates$'
```
Comment (starsirius) [inline config.coffee]: Regex!!!

**Comment by starsirius** [comment]
Merging! Thanks for taking care of this!

**Pull request merged**